.github/workflows/ci.yml

```diff
@@ -27,6 +27,15 @@ jobs:
             echo "$HOME/.elan/bin/" >> $GITHUB_PATH
         if: runner.os == 'Linux'
 
+      - name: Install neovim
+        run: |
+          neovim_dir=$HOME/neovim
+          mkdir $neovim_dir
+          curl -sL https://github.com/neovim/neovim/releases/download/v0.5.0/nvim-linux64.tar.gz \
+            | tar xzf - --strip-components=1 -C $neovim_dir
+          echo $neovim_dir/bin >> $GITHUB_PATH
+        if: runner.os == 'Linux'
```

Comment:
This makes the tests *only* run on 0.5 on Linux -- ideally we'd run both 0.5 and nightly (which is what that github-actions-setup.sh does) if we support both, which is the annoying bit I didn't want to deal with (modifying the matrix to parallelize that).

If you don't want to do the matrix modifying though at least perhaps leave both blocks so they run in serial.

Reply:
The https://raw.githubusercontent.com/norcalli/bot-ci/master/scripts/github-actions-setup.sh only downloads nightlies, so it won't help here.  But I'll add a matrix.

Reply:
This was actually easier than I'd thought, but now we're downloading neovim release binaries on both ubuntu and macos (instead of using brew).  I hope that's ok for you.

Reply:
Nice! That's easier than I thought it'd be too -- yeah that's great, I think the reason I didn't do it that way in the first place was because neovim was in the midst of moving over to GitHub actions a few months ago when I wrote this, so they weren't pushing nightlies, but this makes total sense now.